hack/make.sh

```diff
@@ -183,7 +183,7 @@ bundle() {
 main() {
 	if [ -z "${KEEPBUNDLE-}" ]; then
 		echo "Removing bundles/"
-		rm -rf "bundles/*"
+		rm -rf bundles/*
```

Comment:
nit: why not jut `rm -rf bundles`

Reply:
I _think_ this was done because the `bundles` directory itself is bind-mounted into the container, so we can't remove the directory itself